core-web/apps/dotcms-ui/src/app/portlets/dot-apps/dot-apps-import-export-dialog/dot-apps-import-export-dialog.component.html

```diff
@@ -11,42 +11,48 @@
     <form [formGroup]="form" [ngSwitch]="action" novalidate>
         <ng-container *ngSwitchCase="'Export'">
             <div class="field">
-                <label class="form__label" for="export-password">{{ 'Password' | dm }}</label>
+                <label class="p-label-input-required" for="export-password">{{
```

Comment:
missing unit test

Reply:
Not updating unit tests here, out of the scope. We can set the rule for new unit tests when some bigger refactor happen in the components.